keras_cv/layers/preprocessing/solarization_test.py

```diff
@@ -12,72 +12,71 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import tensorflow as tf
+from absl.testing import parameterized
 
 from keras_cv.layers.preprocessing.solarization import Solarization
 
 
-class SolarizationTest(tf.test.TestCase):
-    def test_range_0_to_255(self):
+class SolarizationTest(tf.test.TestCase, parameterized.TestCase):
+    @parameterized.named_parameters(
+        ("0_255", 0, 255),
```

Comment:
The naming of the test case doesn't seems to carry any meaningful info.

Reply:
how so?  They tell you which values fail.  They get postpended: test_only_values_above_threshold_are_solarized_0_0